internal/awsservice/cloudformation.go

```diff
@@ -42,12 +42,16 @@ func StartStack(ctx context.Context, stackName string, client *cloudformation.Cl
 }
 
 func DeleteStack(ctx context.Context, stackName string, client *cloudformation.Client) {
+	r := recover()
```

Comment:
Should we push stuff out to the terraform level, and use terraform to create the stack, and then have a step after that terraform apply to run the CloudFormation test?

Reply:
I like not needing terraform for our cf test. I think this is simpler for cf but I may be wrong.

Reply:
+1 for being able to run `go test ./abc/def -run xyz` from an EC2 instance and having it test CWA on the local EC2 instance, or in this case execute CFN to launch another instance.